features/support/rspec_before_after_helper.rb

```diff
@@ -0,0 +1,11 @@
+RSpec::Matchers.define :appear_before do |later_content|
+    match do |earlier_content|
+      begin
+        page.body.index(earlier_content) < page.body.index(later_content)
+      rescue ArgumentError
+        raise "Could not locate later content on page: #{later_content}"
+      rescue NoMethodError
+        raise "Could not locate earlier content on page: #{earlier_content}"
+      end
+    end
+  end
```

Comment:
`end` at 11, 2 is not aligned with `RSpec::Matchers.define :appear_before do |later_content|` at 1, 0